tiledb/sm/cpp_api/config.h

```diff
@@ -745,6 +745,20 @@ class Config {
     return val;
   }
 
+  /**
+   * Check if a configuration parameter exists.
+   * @param param Name of configuration parameter
+   * @return true if the parameter exists, false otherwise
+   */
+  bool contains(const std::string& param) const {
+    const char* val;
+    tiledb_error_t* err;
+    tiledb_config_get(config_.get(), param.c_str(), &val, &err);
+    impl::check_config_error(err);
```

Comment:
Since this is supposed to check on existence yes/no, I assume a "no" will be an acceptable (common?) occurrence.  I don't think it would be an error per se, so would recommend removing `check_config_error`.

Reply:
That's true, we expect the `param` will not exist in some cases and want to receive a `bool` result instead of an error. It looks like `check_config_error` would only throw when Config fails the `sanity_check`, but an error message and `false` return value will be enough to handle that rare case. Removing `check_config_error`.